### PR TITLE
refactor: Update chromedriver retrieval

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Now there are separate chromedriver build for M1 mac architecture. Also, the stuff inside of `install.js` mostly duplicates the content of chromedriver storage, so I've refactored it to avoid the unnecessary duplication.

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -19,9 +19,15 @@ const CD_BASE_DIR = path.resolve(__dirname, '..', '..', 'chromedriver');
...
-const MAC_32_ONLY = '2.23.0';
-const LINUX_32_ONLY = '2.34.0';
+const OS = {
+  linux: 'linux',
+  windows: 'win',
+  mac: 'mac'
+};
+const X64 = '64';
+const X86 = '32';
+const M1_ARCH_SUFFIX = '_m1';
```
Comment (KazuCocoa) [inline lib/utils.js:29]: :+1:

File: lib/utils.js [hunk 2]
```diff
@@ -19,9 +19,15 @@ const CD_BASE_DIR = path.resolve(__dirname, '..', '..', 'chromedriver');
 const CD_CDN = process.env.npm_config_chromedriver_cdnurl
   || process.env.CHROMEDRIVER_CDNURL
```
Comment (KazuCocoa) [inline lib/utils.js:20]: Not for this PR. In China, sometimes `https` does not work, but `http` works.  (I learned it recently)
So, this environment variable, `CHROMEDRIVER_CDNURL`, should be here to allow users to configure for their env.

**Review by KazuCocoa — APPROVED** [review]
seems good. I also noticed the `_m1` suffix in the latest stable chromedriver a few days ago. Thanks for this change 👍

File: lib/utils.js (hunk 2 above)
Comment (mykola-mokhnach) [reply lib/utils.js:20]: yeah, the great firewall of China

**Pull request merged**